at_secondary/at_secondary_server/lib/src/verb/handler/change_verb_handler.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:collection';
+
+import 'package:at_commons/at_commons.dart';
+import 'package:at_persistence_spec/src/keystore/secondary_keystore.dart';
+import 'package:at_secondary/src/notification/stats_notification_service.dart';
+import 'package:at_secondary/src/verb/handler/abstract_verb_handler.dart';
+import 'package:at_server_spec/at_server_spec.dart';
+
+/// Responsible for sending the latest commitId to the StatsNotificationService.
+abstract class ChangeVerbHandler extends AbstractVerbHandler {
```

Comment:
Add Dart docs on which verb handlers should extend this. Like ones that generate changes to key store should extend this verb handler etc...

Reply:
Will improve the dart-docs and push the changes